kratos/python/add_mesh_to_python.cpp

```diff
@@ -147,13 +147,25 @@ pybind11::list CalculateOnIntegrationPoints(
     std::vector<TDataType> Output;
     dummy.CalculateOnIntegrationPoints(rVariable, Output, rProcessInfo);
     pybind11::list result;
-    for (unsigned int j = 0; j < Output.size(); j++)
-    {
+    for (std::size_t j = 0; j < Output.size(); j++) {
         result.append(Output[j]);
     }
     return result;
 }
 
+template< class TObject>
+pybind11::list CalculateOnIntegrationPointsBool(
+    TObject& dummy, const Variable<bool>& rVariable, const ProcessInfo& rProcessInfo)
+{
+    std::vector<bool> Output;
+    dummy.CalculateOnIntegrationPoints(rVariable, Output, rProcessInfo);
+    pybind11::list result;
+    for (std::size_t j = 0; j < Output.size(); j++) {
+        result.append(static_cast<int>(Output[j]));
```

Comment:
```suggestion
        result.append(static_cast<bool>(Output[j]));
```

why not like this?

Reply:
Its already bool, the problem is the pybind11::list